File: patch2.py
import os

filepath = 'src/screens/CustomerDashboardScreen.js'
with open(filepath, 'r') as f:
    content = f.read()

target = """        {/* ORDERS SECTION */}
        <Text style={styles.sectionTitle}>Your Active Orders</Text>"""

replacement = """        {/* QUICK ACTIONS */}
        <Text style={styles.sectionTitle}>Quick Actions</Text>
        <QuickActionCard
          title="New Stitch Order"
          subtitle="Upload your design and get it stitched."
          cta="Start Order"
          icon={<Scissors size={24} color={Colors.white} />}
          primary={true}
          onPress={() => { /* Navigate to new stitch order flow */ }}
        />
        <QuickActionCard
          title="Shop Ready-made"
          subtitle="Browse dresses from boutiques."
          cta="Shop Now"
          icon={<ShoppingBag size={24} color={Colors.primary} />}
          onPress={() => navigation.navigate('CustomerShop')}
        />
        <QuickActionCard
          title="My Designs"
          subtitle="View saved inspirations and uploaded designs."
          cta="Open Gallery"
          icon={<Camera size={24} color={Colors.primary} />}
          onPress={() => navigation.navigate('CustomerGallery')}
        />

        {/* ORDERS SECTION */}
        <Text style={[styles.sectionTitle, { marginTop: 24 }]}>Your Active Orders</Text>"""

if target in content:
    content = content.replace(target, replacement)
    with open(filepath, 'w') as f:
        f.write(content)
    print("Success")
else:
    print("Target not found")

File: patch2.js
const fs = require('fs');
const path = 'src/screens/CustomerDashboardScreen.js';
let code = fs.readFileSync(path, 'utf8');

const oldCondition = `(outfit.requestedPhotosFromClient === true || outfit.requestedPhotosFromClient === 'true' || outfit.requestedPhotosFromClient === 1 || String(outfit.requestedPhotosFromClient) === '1' || outfit.requested_photos_from_client === true || outfit.requested_photos_from_client === 'true' || outfit.requested_photos_from_client === 1 || String(outfit.requested_photos_from_client) === '1')`;

const newCondition = `((outfit.requestedPhotosFromClient && outfit.requestedPhotosFromClient !== '0' && outfit.requestedPhotosFromClient !== 'false' && outfit.requestedPhotosFromClient !== 0) || (outfit.requested_photos_from_client && outfit.requested_photos_from_client !== '0' && outfit.requested_photos_from_client !== 'false' && outfit.requested_photos_from_client !== 0))`;

code = code.replace(oldCondition, newCondition);
fs.writeFileSync(path, code);
